README.md

```diff
@@ -117,6 +117,7 @@ _For a complete listing of all available packages, see [Django Packages](https:/
 
 ### Logging
 - [django-guid](https://github.com/JonasKs/django-guid) - Inject a GUID (Correlation-ID) into every log message in a Django request.
+- [django-imgix](https://github.com/imgix/django-imgix) - Transform, optimize, and intelligently cache your entire image library for fast websites and app. Commercial project w/ generous free tier.
```

Comment:
Think you misplaced it? Don't really have anything to do with logging?

Reply:
Ah, whoops! Don't know how it got there. Thanks for pointing this out, I'll fix it now @JonasKs.